notifications/rest/users/delete-user/delete-user.5.x.cs

```diff
@@ -0,0 +1,21 @@
+// Download the twilio-csharp library from www.twilio.com/docs/libraries/csharp#installation
+using System;
+using System.Collections.Generic;
+using Twilio;
+using Twilio.Rest.Notify.V1.Service;
+
+public class Example
+{
+    public static void Main(string[] args)
+    {
+        // Find your Account SID and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+
+        const string serviceSid = "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+        NotifyUserResource.Delete("User001");
```

Comment:
The class name might wind up being `UserResource` instead of `NotifyUserResource`.